sensu/plugins/check-ceph.rb

```diff
@@ -27,10 +27,18 @@
 #   normally considered Ceph warnings to be overlooked and considered
```

Comment:
osd down related test result:
# ./check-ceph-new.rb --criticality critical
CheckCephHealth WARNING: HEALTH_WARN 687 pgs degraded; 684 pgs stuck unclean; 687 pgs undersized; recovery 7670/135927 objects degraded (5.643%); 1/18 in osds are down
# ./check-ceph-new.rb --criticality critical -e 'osds are down,near full osd,clock skew,mons down'
CheckCephHealth CRITICAL: HEALTH_WARN 687 pgs degraded; 685 pgs stuck unclean; 687 pgs undersized; recovery 7670/135927 objects degraded (5.643%); 1/18 in osds are down
# ./check-ceph-new.rb --criticality critical -e 'osds are down,near full osd,clock skew,mons down' -o -f -p
CheckCephHealth CRITICAL: HEALTH_WARN 687 pgs degraded; 685 pgs stuck unclean; 687 pgs undersized; recovery 7670/135927 objects degraded (5.643%); 1/18 in osds are down
ID WEIGHT   TYPE NAME      UP/DOWN REWEIGHT PRIMARY-AFFINITY
-1 15.47974 root default
-2  5.15991     host ceph1
 0  0.85999         osd.0       up  1.00000          1.00000
 1  0.85999         osd.1     down  1.00000          1.00000
 2  0.85999         osd.2       up  1.00000          1.00000
 3  0.85999         osd.3       up  1.00000          1.00000
 4  0.85999         osd.4       up  1.00000          1.00000
 5  0.85999         osd.5       up  1.00000          1.00000
-3  5.15991     host ceph2
 6  0.85999         osd.6       up  1.00000          1.00000
 7  0.85999         osd.7       up  1.00000          1.00000
 8  0.85999         osd.8       up  1.00000          1.00000
 9  0.85999         osd.9       up  1.00000          1.00000
10  0.85999         osd.10      up  1.00000          1.00000
11  0.85999         osd.11      up  1.00000          1.00000
-4  5.15991     host ceph3
12  0.85999         osd.12      up  1.00000          1.00000
13  0.85999         osd.13      up  1.00000          1.00000
14  0.85999         osd.14      up  1.00000          1.00000
15  0.85999         osd.15      up  1.00000          1.00000
16  0.85999         osd.16      up  1.00000          1.00000
17  0.85999         osd.17      up  1.00000          1.00000